packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -113,3 +100,32 @@ class DefaultTaggedText extends StatelessWidget {
     );
   }
 }
+
+/// Creates a bold version of [style].
+TextStyle bold(TextStyle style) => style.copyWith(fontWeight: FontWeight.w600);
```

Comment:
bold is typically w700. Why are we using 600?

Reply:
Full bold all the way down is tough on the eyes.  I've updated the naming for consistency.